security/subsystem/src/main/resources/subsystem-templates/security.xml

```diff
@@ -58,5 +58,21 @@
            </security-domain>
        </replacement>
    </supplement>
-   <supplement name="web-build"/>
+   <supplement name="web-build">
+       <replacement placeholder="ADDITIONAL_SECURITY_DOMAINS">
+           <security-domain name="jboss-web-policy" cache-type="default">
+               <authorization>
+                   <policy-module code="Delegating" flag="required"/>
+               </authorization>
+           </security-domain>
+           <security-domain name="jaspitest" cache-type="default">
```

Comment:
@jamezp Initially, yes, it wasn't in the servlet distribution before, but JASPIC is likely to be used in a servlet distribution. The jaspitest domain is there to allow the activation of JASPIC without having to modify the servlet distribution. Probably it won't be needed in the future but that depends on how Elytron will interact to the security substem. It was crosschecked with @darranl , Elytron team is working in its JASPIC integration, but it is still unclear how both subsystems will interact, so probably we will need the jaspitest there for some time.

Reply:
Should we call it something like `default` rather than `dummy`? Also maybe something better than `jaspitest`. Just seems odd naming to me for a default configuration.

Reply:
@jamezp I agree, it sounds odd to me too. However I'm not sure if it is worth the renaming, as far as I understand it, this security domain will be removed soon. If we rename it, we should update not only the servlet distribution but also the normal distribution and all configuration templates, probably documentation should be updated as well. Said that, should I proceed with the renaming or it is fine to you as it is?

Reply:
The name doesn't matter too much to me really. Just seemed odd :) I'm not even sure what or where it's used so it's your call.

Reply:
Then, I'm fine leaving it as it is now.